app/script/notification/NotificationRepository.js

```diff
@@ -813,4 +811,31 @@ z.notification.NotificationRepository = class NotificationRepository {
     this.notifications.push(notification);
     this.logger.info(`Added notification for ${messageInfo} in '${conversationId}' to queue.`);
   }
+
+  /**
+   * @param {z.entity.Conversation} conversationEntity - The conversation to filter from.
+   * @param {z.entity.Message} messageEntity - The message to filter from.
+   * @param {string} userId - The user id to check mentions for.
+   * @returns {boolean} If the conversation should send a notification.
+   */
+  static _shouldNotify(conversationEntity, messageEntity, userId) {
```

Comment:
No need for an underscore in the name. This is staic and available for everyone.